src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -13,6 +13,8 @@
 #include "ipc/topology.h"
 
 #define GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ 48000
+#define GOOGLE_RTC_AUDIO_PROCESSING_FRAMES_PER_SECOND 100
```

Comment:
this is redefined, should it go in a header?

Reply:
I would prefer to not put it into the header. google_rtc_audio_processing.h and instead redefine it here.